src/e2e/specs/breachResolution.spec.js

```diff
@@ -59,7 +59,7 @@ test.describe('Breach Resolution', () => {
     await expect(async () => {
       await expect(updatedResolvedBreachTab).toBeTruthy()
     }).toPass({
-      timeout: 5000
+      timeout: 2000
```

Comment:
Why do we have magic numbers for the timeout here and below? Ideally, the default timeout suffices for all tests, and if there is a specific reason a particular test takes longer (e.g. a data broker scan is always set to wait a minute before polling for results), that reason is mentioned in a comment next to the magic number.

Reply:
this is added for a quicker fail of the assertion(instead of the default timeout). This is essentially a keep trying every i believe 200ms up until the 2000ms. This allows the script room for any UI/CI weirdness

Reply:
OK, then I'd add that in a comment - though in general, I'd love to see "UI/CI weirdness" addressed the first time it shows up, because that has a habit of accumulating over time and making our tests flaky, while getting harder and harder to debug.